<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RGBLightDetail</class>
 <widget class="QWidget" name="RGBLightDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RGBLightDetail</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>模式</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_constant">
        <property name="text">
         <string>静止</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_blink">
        <property name="text">
         <string>闪烁</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_breath">
        <property name="text">
         <string>呼吸</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_chase">
        <property name="text">
         <string>跑马灯</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_rainbow">
        <property name="text">
         <string>彩虹</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_stream">
        <property name="text">
         <string>流光</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_animation">
        <property name="text">
         <string>动画</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_custom">
        <property name="text">
         <string>音乐律动</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>颜色</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4,4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_color">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(255,255,255);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_pickcolor">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>选择颜色</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,8,1">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>R</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_red">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_red">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>255</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,8,1">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>G</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_green">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_green">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>255</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,8,1">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>B</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_blue">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_blue">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>255</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,9">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>色值</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_color">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>#FFFFFF</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,8,1">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>亮度</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_brightness">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_brightness">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>255</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,8,1">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>色温</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_temperature">
            <property name="minimum">
             <number>1000</number>
            </property>
            <property name="maximum">
             <number>20000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
            <property name="pageStep">
             <number>1000</number>
            </property>
            <property name="value">
             <number>6500</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_temprature">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>6500</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>信息</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_pickcolor</sender>
   <signal>clicked()</signal>
   <receiver>RGBLightDetail</receiver>
   <slot>onPickColorButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_red</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RGBLightDetail</receiver>
   <slot>onRedSliderChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_green</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RGBLightDetail</receiver>
   <slot>onGreenSliderChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_blue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RGBLightDetail</receiver>
   <slot>onBlueSliderChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_color</sender>
   <signal>textEdited(QString)</signal>
   <receiver>RGBLightDetail</receiver>
   <slot>onColorLineEditChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_brightness</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RGBLightDetail</receiver>
   <slot>onBrightnessSliderChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>477</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_temperature</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RGBLightDetail</receiver>
   <slot>onTemperatureSliderChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>506</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_constant</sender>
   <signal>clicked()</signal>
   <receiver>RGBLightDetail</receiver>
   <slot>onConstantButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_blink</sender>
   <signal>clicked()</signal>
   <receiver>RGBLightDetail</receiver>
   <slot>onBlinkButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_breath</sender>
   <signal>clicked()</signal>
   <receiver>RGBLightDetail</receiver>
   <slot>onBreathButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_chase</sender>
   <signal>clicked()</signal>
   <receiver>RGBLightDetail</receiver>
   <slot>onChaseButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>488</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_rainbow</sender>
   <signal>clicked()</signal>
   <receiver>RGBLightDetail</receiver>
   <slot>onRainbowButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>487</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_stream</sender>
   <signal>clicked()</signal>
   <receiver>RGBLightDetail</receiver>
   <slot>onStreamButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_animation</sender>
   <signal>clicked()</signal>
   <receiver>RGBLightDetail</receiver>
   <slot>onAnimationButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>674</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_custom</sender>
   <signal>clicked()</signal>
   <receiver>RGBLightDetail</receiver>
   <slot>onCustomButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>618</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>487</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onPickColorButtonClicked()</slot>
  <slot>onRedSliderChanged(int)</slot>
  <slot>onGreenSliderChanged(int)</slot>
  <slot>onBlueSliderChanged(int)</slot>
  <slot>onColorLineEditChanged(QString)</slot>
  <slot>onBrightnessSliderChanged(int)</slot>
  <slot>onTemperatureSliderChanged(int)</slot>
  <slot>onCustomButtonClicked()</slot>
  <slot>onAnimationButtonClicked()</slot>
  <slot>onStreamButtonClicked()</slot>
  <slot>onRainbowButtonClicked()</slot>
  <slot>onChaseButtonClicked()</slot>
  <slot>onBreathButtonClicked()</slot>
  <slot>onBlinkButtonClicked()</slot>
  <slot>onConstantButtonClicked()</slot>
 </slots>
</ui>
